google-userinfo/background/authorize.js

```diff
@@ -0,0 +1,74 @@
+const REDIRECT_URL = browser.identity.getRedirectURL();
+const CLIENT_ID = "YOUR-CLIENT-ID";
+const SCOPES = ["openid", "email", "profile"];
+const AUTH_URL =
+`https://accounts.google.com/o/oauth2/auth
+?client_id=${CLIENT_ID}
+&response_type=token
+&redirect_uri=${encodeURIComponent(REDIRECT_URL)}
+&scope=${encodeURIComponent(SCOPES.join(' '))}`;
+const VALIDATION_BASE_URL="https://www.googleapis.com/oauth2/v3/tokeninfo";
+
+function extractAccessToken(redirectUri) {
+  let m = redirectUri.match(/[#\?](.*)/);
+  if (!m || m.length < 1)
+    return null;
+  const params = new URLSearchParams(m[1].split("#")[0]);
```

Comment:
Do you mean `params` should be declared with `let`? I don't mean to go against the style guide (although it [doesn't seem to cover this case](https://wiki.mozilla.org/WebExtensions/Hacking#Code_Style)) but it seems more useful to use `const` where we can.